### ISSUE TITLE
Migration of manifest from version 2 to 3

### ISSUE THREAD

**Issue by Ash-KODES** [debkbanerji/netflix-sync-extension#3]

**Comment by debkbanerji** [comment]
Is this an urgent ask given the current Google manifest v2 deprecation timelines?

**Comment by Ash-KODES** [comment]
Yes,because I am working on a same kind of extention and migration in v3 (of a extention having socket.io) is quite complex,beacuse since the service worker  is not persistant we need many workaround to do the same.

**Comment by debkbanerji** [comment]
My question wasn't about the complexity of the migration, but rather the urgency of the migration. The current timeline targets June 2023, which is under review. Apart from the deprecation timeline, are there other reasons for this being an urgent ask, or do you believe that migration work needs to begin soon to hit the target?
